### DISCUSSION TITLE
How to make localGPT use the local model ?

### DISCUSSION THREAD

**Discussion by 50ZAIofficial** [PromtEngineer/localGPT#335] [Q&A]
Some HuggingFace models I use do not have a ggml version. I downloaded the model and converted it to model-ggml-q4.bin through llama.cpp, but I cannot call the model through model_id and model_basename. Am I limited to models from the HuggingFace webpage?
Looking forward to your reply, thank you!

**Comment by 50ZAIofficial** [comment]
I modified run_localGPT.py>load_model>
`model_path = hf_hub_download(repo_id=model_id, filename=model_basename)`
to
`model_path = Your-Model-Path-BinFile`
like
`"C:/Users/X/localGPT/Llama-2-7b-ggml/Llama-2-7b.ggmlv3.q2_K.bin"`
it works well

**Reply by WarmCatUK** [reply]
run_localGPT.py has since changed, and I have the same issue as you. I would like to run a previously downloaded model (mistral-7b-instruct-v0.1.Q8_0.gguf) as I'm currently in a situation where I do not have a fantastic internet connection.
Any ideas?
